### ISSUE TITLE
🐛 Bug Report: Wrong Message When Changing Email on Console

### ISSUE THREAD

**Issue by ItzNotABug** [appwrite/console#975]
### 👟 Reproduction steps

Assuming you have 2 different Email Addresses, each with an Account on Cloud.

---
1. Go to second account
2. Account > Edit Email Address
3. Enter the Email Address of the First account > Enter password > Update
4. Error - `There was an error processing your request. Please check the inputs and try again.`

---
The error should properly depict that the new Email Address already exists on Appwrite or connected to an Organization.

### 👍 Expected behavior

Show proper error message as mentioned above.

### 👎 Actual Behavior

Current shown error is not too clear regarding the actual error.

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by MatthijsDV2004** [comment]
Hi there, I would like to work on this issue, please.

**Comment by MatthijsDV2004** [comment]
Nevermind this bug seems to have been fixed already - should be marked as complete / solved.

**Comment by stnguyen90** [comment]
@ItzNotABug, thanks for creating this issue! 🙏🏼 You mentioned:

> The error should properly depict that the new Email Address already exists on Appwrite or connected to an Organization.

We actually don't want to say the email address already exists because that let's the user know someone has that email on the platform. As such, can this be closed?

**Comment by ItzNotABug** [comment]
@stnguyen90,
The point for not really saying that the email already exists is valid but the current message still seems pretty vague. Even though an email might be in use, the current error says there's an error with the input [presumably frontend form].

We could instead use something or similar like - 
1. `Unable to update email. Please ensure it is correct and not already in use.`
2. `Unable to update email. Please ensure email address is yours and is correctly typed.`

**Comment by stnguyen90** [comment]
@gewenyu99, what are your thoughts on the copy of the error message?

**Comment by gewenyu99** [comment]
@stnguyen90 It's interesting because the solution LinkedIn, Facebook, etc use is actually just a captcha that blocks repeated attempts to snoop for emails.

I really want to be able to display a different response to the Console SDK vs Client/Server SDK.

For for, `unable to update email` is as far as we should go. `Please ensure it is correct and not already in use.` is basically saying it's in use. They can tell if it's a valid email they tried.

**Comment by stnguyen90** [comment]
> I really want to be able to display a different response to the Console SDK vs Client/Server SDK.

@gewenyu99, to do this, the Console needs to check the type (assuming the type is unique enough) and then show some custom message, ignoring the message from the API response.

**Comment by gewenyu99** [comment]
@stnguyen90 Does this API exist for client? Doesn't it require a password? Maybe we just need to rate limit it？

**Comment by stnguyen90** [comment]
> Does this API exist for client? Doesn't it require a password? Maybe we just need to rate limit it？

@gewenyu99, yes, the [update email](https://appwrite.io/docs/references/cloud/client-web/account#updateEmail) endpoint requires a password. There must be a default rate limit on it.

**Comment by eldadfux** [comment]
Starting today this repo is archived. Please raise any issues with the new Appwrite console on the main platform repo: https://github.com/appwrite/appwrite
